packages/block-library/src/button/style.scss

```diff
@@ -36,6 +36,10 @@ $blocks-button__height: 56px;
 	}
 }
 
+.wp-gs .wp-block-button__link:not(.has-background) {
+	background-color: var(--wp-gs-core-button-color-background, var(--wp-gs-color-primary, $dark-gray-700));
```

Comment:
Is `$dark-gray-700` the current default background color for this block? Won't this cause a change in existing themes?

Reply:
Yes, the default is already $dark-gray-700.
https://github.com/WordPress/gutenberg//blob/79d28866952ec3976eafad41d90458aa9d266455/packages/block-library/src/button/style.scss#L17

It will not cause any change in existing themes because they don't use global styles so the class wp-gs is not present.

Reply:
I think we should probably drop $dark-gray-700 from here

Reply:
Hi @youknowriad, what is the reason for removing $dark-gray-700 from here? It is a fact that the default background color will be $dark-gray-700 because of existing rule. But I think setting defaults in this way may open the door to allow tools to easily parse the fallbacks on a given variable and allow the user to easily see the defaults.

Reply:
Today the block doesn't have any default and I feel that's fine. This is changing the behavior of the block for me. 

The question is do we want block authors to provide these defaults like or should we just rely on the theme styles? I feel we always choose the theme styles in general.

For example in dark theme, it doesn't make sense to force a dark color on paragraph text color.